tasks/DependencyCheck/azure-pipelines.yml

```diff
@@ -0,0 +1,56 @@
+name: "0.1.$(rev:r)"
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+  paths:
+    include:
+      - tasks/DependencyCheck/*
+
+pr: none
+
+workspace:
+  clean: all
+
+pool:
+  name: 'DAS - Continuous Integration'
+
+steps:
+- task: Npm@1
+  displayName: 'npm install --production'
+  inputs:
+    command: custom
+    workingDir: '$(System.DefaultWorkingDirectory)/tasks/DependencyCheck/task'
+    verbose: false
+    customCommand: 'install --production'
+- powershell: Remove-Item "$(System.DefaultWorkingDirectory)/tasks/DependencyCheck/task/tests" -Recurse
+  displayName: 'PowerShell Script: Remove tests folder'
+- script: tsc
+  workingDirectory: $(System.DefaultWorkingDirectory)/tasks/DependencyCheck/task
+  displayName: 'Command Line Script: tsc'
+- powershell: |
+   $FileName = "$(System.DefaultWorkingDirectory)/tasks/DependencyCheck/task/node_modules/csvtojson/test/data/data#139"
+   if (Test-Path $FileName)
+   {
+     Remove-Item $FileName
+   }
```

Comment:
Might be good to PR this out of their repo if it affects our build.

Reply:
Agreed, PR made: https://github.com/Keyang/node-csvtojson/pull/367